content/source/docs/enterprise-beta/api/run.html.md

```diff
@@ -10,82 +10,12 @@ sidebar_current: "docs-enterprise2-api-run"
 
 Performing a run on a new configuration is a multi step process.
 
-1. Create a configuration version on the workspace
-2. Upload configuration files to the configuration version
-3. Create a Run on the workspace
-4. Create and queue a plan on the run; this is done automatically when the run is created
-5. Create and queue an apply on the run
+1. [Create a configuration version on the workspace](./configuration-versions.html#create-a-configuration-version)
+2. [Upload configuration files to the configuration version](./configuration-versions.html#upload-configuration-files)
+3. [Create a Run on the workspace](#create-a-run)
```

Comment:
Wait... I thought the configuration-versions page said you didn't have to create a run,  once you'd uploaded a config tarball?? Except that doesn't seem to make sense either, because the response from uploading a file doesn't seem to include a run ID, so there's no way to know which run you would be applying?

Reply:
When a config file is uploaded to the configuration-version it DOES automatically create a run. I wanted to add it to this list so that it is clear what the sequence is. As such, I added the note to explain that it is performed automatically.